common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -239,13 +246,23 @@ const SearchTabs: FunctionComponent<Props> = ({
             more.
           </Space>
           <SearchForm
+            query={query}
+            sort={undefined}
+            sortOrder={undefined}
+            linkResolver={(params) => {
+              const query = propsToQuery(params);
```

Comment:
should this use `imagesLink` or similar?

Reply:
For now I've just left it as URLParams.

Reply:
Actually, changed tact
https://github.com/wellcomecollection/wellcomecollection.org/pull/6101/files#diff-57484c78334523af4b4550d1f0423cb8395ef49a3ad6a67d4eb36333abfa8788R178

https://github.com/wellcomecollection/wellcomecollection.org/pull/6101/files#diff-57484c78334523af4b4550d1f0423cb8395ef49a3ad6a67d4eb36333abfa8788R178

Not the tidiest, but works.